### PR TITLE
Added rabbitmq dependency to chart

### PR DESCRIPTION
**Pull request by erezmus**

### CODE REVIEW

**Review by diversemix — APPROVED** [review]

File: helm/libero-reviewer/requirements.lock [hunk 1]
```diff
@@ -2,5 +2,8 @@ dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
   version: 6.2.1
-digest: sha256:b1791c8964a84e7e17d008a26bd2ab5f6e49e167107746d048bdab25ed078569
-generated: 2019-09-16T15:51:54.715208129+01:00
+- name: rabbitmq
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: 6.12.1
```
Comment (giorgiosironi) [inline helm/libero-reviewer/requirements.lock:7]: this is the chart version, pin the RabbitMQ version with `image.tag` chosen from https://hub.docker.com/r/bitnami/rabbitmq/tags

Comment (giorgiosironi) [inline helm/libero-reviewer/requirements.lock:7]: `persistence.enabled` is `true` by default, do you want that? Would check it's consistent with the `postgresql` value

Comment (giorgiosironi) [inline helm/libero-reviewer/requirements.lock:7]: do you need `rabbitmq.username` and/or `rabbitmq.password` as values?

Comment (giorgiosironi) [reply helm/libero-reviewer/requirements.lock:7]: all documentation is at https://github.com/helm/charts/tree/master/stable/rabbitmq

**Pull request merged**